fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/AsyncWebServiceRequestHandlerSpec.groovy

```diff
@@ -42,6 +42,7 @@ class AsyncWebServiceRequestHandlerSpec extends Specification {
         1 * dws.postDruidQuery(rc, _, null, null, groupByQuery) >> { a0, a1, a2, a3, a4 ->
             // Save the success callback
             sc = a1
+            return null
```

Comment:
Ahh, right, so it was looking for something that it could call a `Future`. Whatever was being returned with just `sc = a1` wasn't castable, but `null` is. So, returning a `Mock(Future<Response>)` is probably the best thing there. (a bit better than `null` at least)